AlfrescoApp/App Delegate/AppDelegate.m

```diff
@@ -108,11 +108,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
             [hockeyManager.authenticator authenticateInstallation];
         }
         
-        // Flurry Analytics
-        if (FLURRY_API_KEY.length > 0)
-        {
-            [[AnalyticsManager sharedManager] startAnalytics];
-        }
```

Comment:
See [IOS-514](https://issues.alfresco.com/jira/browse/IOS-514) - we'll run Flurry in parallel until we're happy GA is recording to a similar accuracy.